bundle/edu.gemini.itc.web/src/main/java/edu/gemini/itc/web/html/HtmlPrinter.java

```diff
@@ -67,69 +60,53 @@ public static String printParameterSummary(final SourceDefinition sdp) {
     }
 
     public static String printParameterSummary(final ObservationDetails odp) {
-        StringBuffer sb = new StringBuffer();
-
-        // This object is used to format numerical strings.
-        FormatStringWriter device = new FormatStringWriter();
-        device.setPrecision(2);  // Two decimal places
-        device.clear();
+        final StringBuilder sb = new StringBuilder();
 
         sb.append("Calculation and analysis methods:\n");
-        sb.append("<LI>mode: " + (odp.getMethod().isImaging() ? "imaging" : "spectroscopy") + "\n");
+        sb.append("<LI>mode: ");
+        sb.append((odp.getMethod().isImaging() ? "imaging" : "spectroscopy"));
+        sb.append("\n");
         sb.append("<LI>Calculation of ");
         if (odp.getMethod().isS2N()) {
-            sb.append("S/N ratio with " + odp.getNumExposures() + " exposures of " + device.toString(odp.getExposureTime()) + " secs,");
-            sb.append(" and " + device.toString(odp.getSourceFraction() * 100) + " % of them were on source.\n");
+            sb.append(String.format("S/N ratio with " + odp.getNumExposures() + " exposures of %.2f secs,", odp.getExposureTime()));
+            sb.append(String.format(" and %.2f %% of them were on source.\n", odp.getSourceFraction() * 100));
         } else {
-            sb.append("integration time from a S/N ratio of " + device.toString(odp.getSNRatio()) + " for exposures of");
-            sb.append(" " + device.toString(odp.getExposureTime()) + " with " + device.toString(odp.getSourceFraction() * 100) + " % of them were on source.\n");
+            sb.append(String.format("integration time from a S/N ratio of %.2f for exposures of", odp.getSNRatio()));
+            sb.append(String.format(" %.2f with %.2f %% of them were on source.\n", odp.getExposureTime(), odp.getSourceFraction() * 100));
         }
         sb.append("<LI>Analysis performed for aperture ");
         if (odp.isAutoAperture()) {
             sb.append("that gives 'optimum' S/N ");
         } else {
-            sb.append("of diameter " + device.toString(odp.getApertureDiameter()) + " ");
+            sb.append(String.format("of diameter %.2f ", odp.getApertureDiameter()));
         }
-        sb.append("and a sky aperture that is " + device.toString(odp.getSkyApertureDiameter()) + " times the target aperture.\n");
+        sb.append(String.format("and a sky aperture that is %.2f times the target aperture.\n", odp.getSkyApertureDiameter()));
 
         return sb.toString();
     }
 
     public static String printParameterSummary(final PlottingDetails pdp) {
-        StringBuffer sb = new StringBuffer();
-
-        // This object is used to format numerical strings.
-        FormatStringWriter device = new FormatStringWriter();
-        device.setPrecision(2);  // Two decimal places
-        device.clear();
+        final StringBuilder sb = new StringBuilder();
 
         sb.append("Output:\n<LI>Spectra ");
         switch (pdp.getPlotLimits()) {
```

Comment:
Do we really need a switch when this is really just overcomplicating an `if`...`else` or even a ternary expression?

(Sorry if any of these have repeated... I thought I made a mistake.)